presto-docs/src/main/sphinx/release/release-346.md

```diff
@@ -0,0 +1,60 @@
+# Release 345 (XX Nov 2020)
+
+## General Changes
+
+* Add support for `RANGE BETWEEN <value> PRECEDING AND <value> FOLLOWING` window frames. ({issue}`609`)
+* Add support for {func}`extract` with `TIMEZONE_HOUR` and `TIMEZONE_MINUTE` for `time with time zone` values. ({issue}`5668`)
+* Add SQL syntax for `GRANT` and `REVOKE` on schema. This is not yet used by any connector. ({issue}`4396`)
+* Add SQL syntax for `ALTER TABLE SET AUTHORIZATION` to change the table owner. ({issue}`5717`)
+* Fix incorrect handling of negative offsets for the `time with time zone` type. ({issue}`5696`)
+* Fix incorrect result when casting `time(p)` to `timestamp(p)` for precisions higher than 6. ({issue}`5736`)
```

Comment:
You may want to include https://github.com/prestosql/presto/issues/5678#issuecomment-721616202 next to this.